### PR TITLE
Edit consumergroup offsets

### PR DESCRIPTION
**Pull request by ognjen-j**

### CODE REVIEW

File: esque/cli/output.py [hunk 1]
```diff
@@ -125,11 +125,11 @@ def pretty_duration(value: Any, *, multiplier: int = 1) -> str:
...
     # Fix for conversion errors of ms > C_MAX_INT in some internal lib
-    if value > C_MAX_INT:
-        value = int(value / MILLISECONDS_PER_YEAR)
+    try:
+        value = int(float(value) / MILLISECONDS_PER_YEAR)
         return pendulum.duration(years=value).in_words()
-
-    return pendulum.duration(milliseconds=value).in_words()
+    except OverflowError:
+        return "unlimited"
```
Comment (swenzel) [inline esque/cli/output.py]: If value is not a time range of multiple years, this function should still return `pendulum.duration(milliseconds=value).in_words()`. We might want to add a few unit tests so this is properly covered.

File: esque/cli/output.py [hunk 2]
```diff
@@ -125,11 +125,11 @@ def pretty_duration(value: Any, *, multiplier: int = 1) -> str:
     value *= multiplier
 
     # Fix for conversion errors of ms > C_MAX_INT in some internal lib
-    if value > C_MAX_INT:
-        value = int(value / MILLISECONDS_PER_YEAR)
+    try:
+        value = int(float(value) / MILLISECONDS_PER_YEAR)
```
Comment (swenzel) [inline esque/cli/output.py]:
```suggestion
        value //= MILLISECONDS_PER_YEAR
```

File: esque/cli/output.py (hunk 1 above)
Comment (ognjen-j) [reply esque/cli/output.py]: I think this should do it.

File: esque/cli/output.py (hunk 2 above)
Comment (ognjen-j) [reply esque/cli/output.py]: Used the divmod function.

**Review by swenzel — APPROVED** [review]
Some ideas for future enhancement: We could support special offset values. Something that allows the user to set the offset to beginning, end or a specific time or to shift it by a specific offset or duration?

**Pull request merged**